### ISSUE TITLE
hugepages: cannot open /sys/devices/system/node/node0/hugepages/hugepages-1048576kB/demote

### ISSUE THREAD

**Issue by ConiKost** [collectd/collectd#3993]
*   Version of collectd: 5.12.0
*   Operating system / distribution: Gentoo
*   Kernel version (if applicable): 5.17.1

## Expected behavior
Do not try to open demote, as it seems write only, but no read.

## Actual behavior
Collectd tries to read /sys/devices/system/node/node0/hugepages/hugepages-1048576kB/demote, but it's a write-only file.

```
$ ls -la /sys/devices/system/node/node0/hugepages/hugepages-1048576kB/demote
--w------- 1 root root 4096  1. Apr 23:10 /sys/devices/system/node/node0/hugepages/hugepages-1048576kB/demote
```

## Steps to reproduce

*   Enable hugepages
*   Start Collectd
*   Check logs

**Comment by lnatapov** [comment]
Any update on this issue ? Was it a misconfiguration ?

**Comment by ConiKost** [comment]
No. This is not a missconfiguration. As the interface changed, the plugin needs to be updated.
